java/gui/src/main/java/com/squareup/subzero/framebuffer/Screens.java

```diff
@@ -195,9 +195,10 @@ public String promptPasswordChange() throws IOException {
     return promptPassword("Please confirm the password");
   }
 
-  public enum ExitOrRestart {
+  public enum DisplayReturn {
```

Comment:
renaming this to make a little more sense. Better name suggestions welcome.

Reply:
Would `ExitOrRestartOrPoweroff` be better (for the explicitness)?

Reply:
done.